Session/Utilities/MentionUtilities.swift

```diff
@@ -110,6 +110,7 @@ public enum MentionUtilities {
                 result.addAttribute(.currentUserMentionBackgroundCornerRadius, value: (8 * sizeDiff), range: mention.range)
                 result.addAttribute(.currentUserMentionBackgroundPadding, value: (3 * sizeDiff), range: mention.range)
                 result.addAttribute(.currentUserMentionBackgroundColor, value: primaryColor.color, range: mention.range)
+                result.addAttribute(.kern, value: (3 * sizeDiff), range: NSRange(location: mention.range.upperBound, length: 1))
```

Comment:
Do we need to do the opposite for RTL languages?

Reply:
We don't need to do the opposite for RTL. If it is RTL, the padding will be added on the left instead of right.

Reply:
Ok awesome, thanks for confirming